### PR TITLE
Expression controllers

### PR DESCRIPTION
**Pull request by alexwbi**

### CODE REVIEW

File: swagger_server/__main__.py [hunk 1]
```diff
@@ -18,15 +19,33 @@
 # ROUTES!
 # ------------------------------------------------------------------------------
 
+
 @app.route('/expression', methods=['POST'])
 def post_expression_view():
-    return add_expression()
+    try:
+        return add_expression(), 200
```
Comment (dsubak) [inline swagger_server/__main__.py:30]: I think you'll need to pass in the post body to `add_expression`

File: swagger_server/__main__.py [hunk 2]
```diff
@@ -18,15 +19,33 @@
...
+
 
 @app.route('/expression/<int:expression_id>', methods=['GET'])
 def get_expression_view(expression_id):
     return get_expression(expression_id)
 
 
+@app.route('/expression/<int:expression_id>', methods=['PUT'])
+def put_expression_view(expression_id):
+    return put_expression(expression_id)
```
Comment (dsubak) [inline swagger_server/__main__.py]: I think `put_expression` should take both `expression_id` and the payload (i.e. request.get_json)

File: swagger_server/controllers/expression_controller.py [hunk 3]
```diff
@@ -9,11 +13,16 @@ def add_expression():  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    try:
+        Expression.create()
+    except Exception as exc:
+        raise ValidationError(400, 'Expression was not created')
+
+    return flask.jsonify(True)
```
Comment (dsubak) [inline swagger_server/controllers/expression_controller.py]: Let's hand back the created Expression's ID here

File: swagger_server/controllers/expression_controller.py [hunk 4]
```diff
@@ -35,7 +46,7 @@ def get_expression(expression_id):  # noqa: E501
 
     :rtype: None
     """
-    return 'do some magic!'
+    return Expression.get_by_id(expression_id)
```
Comment (dsubak) [inline swagger_server/controllers/expression_controller.py]: I've been jsonifying these - I don't think it'll serialize directly w/o something fancy or unpacking em manually.

**Review by chrisconlon-klaviyo — APPROVED** [review]

**Pull request merged**